### ISSUE TITLE
allow changing how fast a "climbable" node is

### ISSUE THREAD

**Issue by pandaro** [minetest/minetest#1935]
I want to suggest this possibility:
change the definition of a node in this way:

climbable = 0.0: this node is not climbable
climbable = 0.5: this node is climbableat at half speed
climbable = 1.0: this node is climbable at normal speed
climbable = 2.0: this node is climbable at double speed
etc...
so it would be possible to make climbable very slowly even the rocky walls.
I do not think something unlikely, because climbing is a sport.
Also those unwilling to climb the walls should just leave things as they are.
I do not know if this feature is implementable, thanks anyway for your attention.

**Comment by paramat** [comment]
:-1: Idea is fine but i don't think worth doing.

**Comment by SmallJoker** [comment]
Different climbing speeds add variety to the game, so we should leave this open. Adding this feature to the localplayer code isn't difficult but anticheat would need a few more checks against ladders nearby.
@paramat, that issue was opened in MTG: minetest/minetest_game#1600.

**Comment by paramat** [comment]
Ok, so are you interested in coding it sometime?

**Comment by SmallJoker** [comment]
@paramat Yes but I'm not quite sure about the  anticheat part. Will have to look into it.

**Comment by farooqkz** [comment]
I think this still makes sense as of 2026. Would someone add it? :)
